dolphinscheduler-dist/release-docs/licenses/LICENSE-hive-shims-0.23.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
These are standard Apache 2.0 so we don't need these copies, please modify https://github.com/apache/dolphinscheduler/blob/a8aed0f5f7d0c8fd2d8acf04359c9cff8276df14/dolphinscheduler-dist/release-docs/LICENSE#L211-L225